src/test_ string_to_array.py

```diff
@@ -0,0 +1,43 @@
+"""Testing solutions for CodeWars Problem string to array()."""
+
+# These are the tests from code wars
+
+# Test.describe("Basic tests")
+# Test.assert_equals(string_to_array("Robin Singh"), ["Robin", "Singh"])
+# Test.assert_equals(string_to_array("CodeWars"), ["CodeWars"])
+# Test.assert_equals(string_to_array("I love arrays they are my favorite"), ["I", "love", "arrays", "they", "are", "my", "favorite"])
+# Test.assert_equals(string_to_array("1 2 3"), ["1", "2", "3"])
+# Test.assert_equals(string_to_array(""), [""])
+
+
+# These are my own tests
+
+
+def test_string_to_array_name():
```

Comment:
This (and your other test files apparently) would've been a great time to use `@pytest.mark.parametrize`